src/components/attribute-picker/attribute-picker-item.js

```diff
@@ -4,22 +4,13 @@ import '@brightspace-ui/core/components/tooltip/tooltip.js';
 import { css, html, LitElement } from 'lit';
 import { bodyCompactStyles } from '@brightspace-ui/core/components/typography/styles.js';
 import { LocalizeLabsElement } from '../localize-labs-element.js';
-import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 
-class AttributePickerItem extends RtlMixin(LocalizeLabsElement(LitElement)) {
+class AttributePickerItem extends LocalizeLabsElement(LitElement) {
 	static get properties() {
 		return {
-			deletable: {
-				type: Boolean,
-				reflect: true
-			},
-			tabIndex: {
-				type: Number,
-				reflect: true
-			},
-			text: {
-				type: String
-			}
+			deletable: { type: Boolean, reflect: true },
+			tabIndex: { type: Number, reflect: true },
```

Comment:
`tabIndex` is a native thing that all elements have, so you shouldn't need to redefine it as a Lit reactive property _unless_ you need the act of changing it to trigger a re-render.

Reply:
Without it in properties, the `runConstructor` actually fails, which is why I'd initially done it, but I move the initialization of it to `firstUpdated`, then it actually works as expected, so I'll make that change 👍